templates/common/_base/files/mtu-migration-conf.yaml

```diff
@@ -0,0 +1,22 @@
+{{ $target_mtu := 0 -}}
+{{ $cni_target_mtu := 0 -}}
+{{ if .MTUMigration -}}
+{{ if .MTUMigration.Machine }}{{ if .MTUMigration.Machine.To -}}
+{{ $target_mtu = .MTUMigration.Machine.To -}}
+{{ end }}{{ end -}}
+{{ if .MTUMigration.Network }}{{ if .MTUMigration.Network.To -}}
+{{ $cni_target_mtu = .MTUMigration.Network.To -}}
+{{ end }}{{ end -}}
+{{ end -}}
+{{ if or $target_mtu $cni_target_mtu -}}
```

Comment:
In the MCO today we don't really have the concept of a empty file definition. Would it be acceptable if the file was always defined but has empty contents when not used?

Could you remind me what is consuming this file?

Reply:
This file will be used by a new systemd service (PR pending) that will configure host interfaces MTU during migration. Yes, we can have an empty file instead I guess and enable/disable the systemd service conditionally.

Reply:
Although we will probably be also be deploying a NM dispatcher script that is only of use during a migration and it would be a pity that is running all the time. I guess there is other ways we could work around this.